compiler/noirc_frontend/src/tests/cfg.rs

```diff
@@ -0,0 +1,506 @@
+#![cfg(test)]
```

Comment:
The tests here could use the latest way to test errors that use "^^^" and "~~~".